src/registrar/templates/home.html

```diff
@@ -41,7 +41,16 @@ <h2>Domains</h2>
           <th data-sortable scope="col" role="columnheader">Domain name</th>
           <th data-sortable scope="col" role="columnheader">Expires</th>
           <th data-sortable scope="col" role="columnheader">Status</th>
-          <th scope="col" role="columnheader"><span class="usa-sr-only">Action</span></th>
+          <th 
+            scope="col" 
+            role="columnheader" 
+            {% if has_deletable_applications %} class="action-col-custom-width" {% else %} class="width-15" {% endif %}
```

Comment:
Why is this needed?

Reply:
By default, the "manage" button doesn't align with the manage button in the other table. This is especially apparent if the user doesn't have anything that they can delete, so there is just an empty column. ` has_deletable_applications` is a flag that switches between the default table display (as it was before this change w/o that column) and the new one. These two scenarios need different widths on the manage button, especially because it wasn't correctly aligned before this PR